auth_mellon_config.c

```diff
@@ -1099,6 +1099,14 @@ const command_rec auth_mellon_commands[] = {
         " '/'."
         ),
     AP_INIT_TAKE1(
+        "MellonCookieSameSite",
+        ap_set_string_slot,
```

Comment:
I think this option needs a custom parser, along the lines of the one for the [`MellonEnable`-option](https://github.com/UNINETT/mod_auth_mellon/blob/v0.12.0/auth_mellon_config.c#L426). The reason for that is that if the user makes a mistake when specifying this option, e.g. by misspelling "strict", the user will get no feedback that something is wrong. Instead, the option will be silently ignored.